en/firefox/facebook_container.ftl

```diff
@@ -12,7 +12,7 @@ facebook-container-download-firefox-and-get-the = Download { -brand-name-firefox
 facebook-container-only-available-for-desktop = The { -brand-name-facebook-container } Extension is currently only available for { -brand-name-firefox } for Desktop.
 facebook-container-brand-name-firefox-browser = { -brand-name-firefox-browser }
 # Obsolete string
-facebook-container-firefox-browser = { -brandname-firefox-browser }
+facebook-container-firefox-browser = { -brand-name-firefox-browser }
```

Comment:
Because of the change, we need to add a `-v2` to the ID to trigger the change, even though this is an obsolete string.

Reply:
oops, sorry